userguide/advanceduse/screencasting.rst

```diff
@@ -0,0 +1,34 @@
+Screen Casting your UT device to your computer
+==============================================
+
+
+Ubuntu Touch comes with a command line utility called ``mirscreencast`` which dumps screen frames to a file.
+The idea here is to stream UT display to a listening computer over the network so that we can either watch it live or record it to a file.
+
+On computer side:
+-----------------
+  For real time casting:
+
+  Prepare your computer to listen to a tcp port(here 1234) and forward stream to a video player (here mplayer):
+
+      nc -l -p 1234 | gzip -dc | mplayer -demuxer rawvideo -rawvideo w=384:h=640:format=rgba -
```

Comment:
Use two colons followed by an indented line to create a code block. For example...

```
[...]and forward stream to a video player (here mplayer)::

    nc -l[...]
```